modules/swagger-codegen/src/main/resources/dart/pubspec.mustache

```diff
@@ -4,7 +4,7 @@ description: {{pubDescription}}
 dependencies:
   http: '>=0.11.1 <0.12.0'
   dartson: "^0.2.4"
-  intl: "^0.12.4+2"
+  intl: any
```

Comment:
What about using `">=0.12.4"` instead of `any` to ensure a minimal version is installed?